staging/cse/windows/kubeletfunc.ps1

```diff
@@ -209,6 +209,70 @@ function Get-KubePackage {
     Remove-Item $zipfile
 }
 
+function Remove-KubeletNodeLabel {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $KubeletNodeLabels,
+        [Parameter(Mandatory=$true)][string]
+        $Label
+    )
+
+    if ($KubeletNodeLabels -Match ",$Label") {
+        return $KubeletNodeLabels -replace ",$Label", ""
+    }
+
+    if ($KubeletNodeLabels -Match "$Label,") {
+        return $KubeletNodeLabels -replace "$Label,", ""
+    }
+
+    if ($KubeletNodeLabels -Match "$Label") {
+        return $KubeletNodeLabels -replace "$Label", ""
+    }
+
+    return $KubeletNodeLabels
+}
+
+function Get-TagValue {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $TagName,
+        [Parameter(Mandatory=$true)][string]
+        $DefaultValue
+    )
+
+    $uri = "http://169.254.169.254/metadata/instance?api-version=2021-02-01"
+    $response = Retry-Command -Command "Invoke-RestMethod" -Args @{Uri=$uri; Method="Get"; ContentType="application/json"; Headers=@{"Metadata"="true"}} -Retries 3 -RetryDelaySeconds 5

```

Comment:
We may need to add error code if all retires fail

Reply:
right now we just default to returning false if all retries fail which I personally think is fine, though I can add an error code if you think that would be better

Reply:
I added a try/catch, where we just return the default value if an error occurs - let me know what you think

Reply:
in the end I went with a new exit code to align with the linux implementation